dstep/translator/HeaderIndex.d

```diff
@@ -170,98 +223,170 @@ class HeaderIndex
 {
     private IncludeGraph includeGraph_;
     private string[string] stdLibPaths;
+    private string[string] knownModules;
     private string mainFilePath;
 
     public this(TranslationUnit translationUnit)
     {
+        immutable string[string] standardModuleMapping = [
```

Comment:
Shouldn't this be moved out of the constructor as a constant. The same associative array will be created each time the constructor is called, unless the compiler is doing something smart.

Reply:
I tried, but D is too dumb for this : (:

```
dstep/translator/HeaderIndex.d(222,50): Error: non-constant expression `["assert.h":"core.stdc.assert_", 
```

Reply:
Hmm, right. One would have to use a shared module constructor for this.

Reply:
Lets keep it simple. It will be called only once per translation unit.